app/src/main/java/org/wikipedia/feed/accessibility/AccessibilityCardView.kt

```diff
@@ -0,0 +1,45 @@
+package org.wikipedia.feed.accessibility
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.LinearLayout
+import org.wikipedia.databinding.ViewCardAccessibilityBinding
+import org.wikipedia.feed.model.Card
+import org.wikipedia.feed.view.FeedAdapter
+import org.wikipedia.feed.view.FeedCardView
+
+class AccessibilityCardView(context: Context?) : LinearLayout(context), FeedCardView<Card?> {
+
+    private var _binding: ViewCardAccessibilityBinding? = null
+    private val binding get() = _binding!!
```

Comment:
You can inflate the layout directly here without having an extra getter.

Reply:
@cooltey The getter is created everywhere to manage the lifecycle, just as an extra precaution. I say we keep it both for safety and consistency..